modules/markup/html.go

```diff
@@ -99,7 +99,7 @@ var issueFullPatternOnce sync.Once
 func getIssueFullPattern() *regexp.Regexp {
 	issueFullPatternOnce.Do(func() {
 		issueFullPattern = regexp.MustCompile(regexp.QuoteMeta(setting.AppURL) +
-			`\w+/\w+/(?:issues|pulls)/((?:\w{1,10}-)?[1-9][0-9]*)([\?|#](\S+)?)?\b`)
+			`[\w_.-]+/[\w_.-]+/(?:issues|pulls)/((?:\w{1,10}-)?[1-9][0-9]*)([\?|#](\S+)?)?\b`)
```

Comment:
```suggestion
			`[\w_\.-]+/[\w_\.-]+/(?:issues|pulls)/((?:\w{1,10}-)?[1-9][0-9]*)([\?|#](\S+)?)?\b`)
```
`.` matches every character.

Reply:
Oh, good point, thanks!

Reply:
This is incorrect, inside character classes, `.` (and all other special characters) have no special meaning, so this this qualifies as a useless escaping.

If you attempt the same regex in JS, the linter would [not like it](https://eslint.org/docs/rules/no-useless-escape), so I would revert this change.

Reply:
Ah, I see it was fixed already.